testgrid/build_test_update.md

```diff
@@ -5,15 +5,15 @@ If you need to add a new test that you want TestGrid to display, see [TestGrid C
 If you're looking to develop for TestGrid, welcome! Note that most of the inner workings of TestGrid
 are not open source. This is a [known issue](https://github.com/kubernetes/test-infra/issues/10409).
 
-## config.yaml and config.proto
+## YAML configuration and config.proto
 
 While TestGrid configuration is located in YAML files, TestGrid doesn't natively
 read YAML. Instead, it expects configuration data in [`config.proto`]. This file
 is commented, and should be treated as the authoritative "input" schema to
 TestGrid.
 
 [`config.proto`] is generated primarily with [Configurator](cmd/configurator).
-Updates to [`config.yaml`] are automatically Configurated when a change is
+Updates to the [k8s configuration]( ../config/testgrids) are automatically Configurated when a change is
```

Comment:
I find myself wanting to say "configs" or "config files" more than configuration but trying to stick with your wording for consistency
```suggestion
Updates to the [testgrid.k8s.io configurations]( ../config/testgrids) are automatically Configurated when a change is
```